panopticum/templates/page/component.html

```diff
@@ -184,20 +191,25 @@
 }
 
 function update_component_page_elements(component) {
-
     pa_replace_urls();
 
     $(".pa-quality").each(function(index, el) {
-        $(el).html(pa_star_rating($(el).html()));
-    });
-
-    $(".pa-status").each(function(index, el) {
-        $(el).html(pa_no_partial_yes($(el).html()));
```

Comment:
seems like this is a conflict with my patch

Reply:
I will revert. I does not use that